File: Bug_Hunt_Practice/Question 7/solution.py
def find_first_a(s):
    found_index = -1
    
    for i in range(len(s)):
        if s[i] == 'a':
            found_index = i
            
    return found_index

text = "banana"
index = find_first_a(text)

print(f"Expected Output: 1")
print(f"Actual Output:   {index}")

File: Bug_Hunt_Practice/Question 7/solution.c
#include <stdio.h>
#include <string.h>

int findFirstA(const char* str) {
    int foundIndex = -1;
    int len = strlen(str);
    
    for (int i = 0; i < len; i++) {
        if (str[i] == 'a') {
            foundIndex = i;
        }
    }
    return foundIndex;
}

int main() {
    const char* text = "banana";
    int index = findFirstA(text);
    printf("Expected Output: 1\n");
    printf("Actual Output:   %d\n", index);
    return 0;
}